cmd/podman/images/build.go

```diff
@@ -183,12 +183,6 @@ func buildFlags(cmd *cobra.Command) {
 	completion.CompleteCommandFlags(cmd, fromAndBudFlagsCompletions)
 	flags.SetNormalizeFunc(buildahCLI.AliasFlags)
 	if registry.IsRemote() {
-		flag = flags.Lookup("isolation")
```

Comment:
Does this continue to work in non rootless mode?

Reply:
yes the default is handled by buildah just like the other flags